code/modules/organs/organ.dm

```diff
@@ -446,6 +446,8 @@ var/global/list/ailment_reference_cache = list()
 
 /obj/item/organ/proc/get_possible_ailments()
 	. = list()
+	if(owner.status_flags & GODMODE)
+		return .
```

Comment:
`Return .` is clearer I would argue

Reply:
I swear I saw a post on the byond forums about this being optimized to be the same bytecode

Reply:
That seems to be incorrect. `TICKBENCH` benchmarking appears to show that `return .` is reliably faster than `return` by about 4% to 6%. It's not enough to be worth splitting hairs over, but `return` is definitely not the faster one.

Reply:
https://www.byond.com/forum/post/2787448

Reply:
https://www.byond.com/forum/post/2787448

Reply:
Honestly the base idea behind just putting return alone returning something that's not null is kind of confusing and error prone. 
And since it's exactly the same whether you put ``.`` or not, I just put it. It's extremely handy to remind yourself that it's not going to return null, and prevent yourself from doing something stupid. 
True story. X_X